src/i18n/langResources/lang.de.ts

```diff
@@ -27,6 +27,7 @@ export const langDe: ILangResource = {
   tsOfficialPresetMessage: 'Offizielles TS Icon wird',
   jsonOfficialPresetMessage: 'Offizielles JSON Icon wird',
   hideFoldersPresetMessage: 'Ordner Icons werden',
+  foldersAllDefaultIconPresetMessage: 'Ordner Icons Auflegemaske werden',
```

Comment:
I'm not quite sure what this feature will offer, correct me if I'm wrong: With this setting it's possible to deactivate any custom folder icon, instead, it will show the normal folder icon?